tide_site.module

```diff
@@ -234,6 +236,56 @@ function tide_site_views_pre_view(ViewExecutable $view, $display_id, array &$arg
     ];
     $view->addHandler($display_id, 'field', 'media__field_media_site', 'field_media_site', $site_field, 'field_media_site');
   }
+  if ($view->id() == 'content' && $view->current_display == 'page_1') {
+    if (_has_target_site_fields()) {
+      $fields_order = [
+        'title',
+        'field_node_primary_site',
+        'field_node_site',
+        'name',
+        'type',
+        'moderation_state',
+        'changed',
+        'operations',
+      ];
+      $filters_order = [
+        'title',
+        'status_extra',
+        'field_node_primary_site_target_id',
+        'field_node_site_target_id',
+        'uid',
+        'type',
+        'moderation_state',
+        'status',
+      ];
+      $module_handler = Drupal::moduleHandler();
+      $configs = Yaml::parseFile($module_handler->getModule('tide_site')->getPath() . '/yaml/content_summary_page.yml');
+      $fields = $view->display_handler->getOption('fields');
+      $exposed_form = $view->display_handler->getOption('exposed_form');
+      $filters = $view->display_handler->getOption('filters');
+      unset($fields['status']);
+      unset($fields['changed']);
+      unset($fields['node_bulk_form']);
```

Comment:
@vincent-gao @anthony-malkoun @GROwen is it necessary to hide the bulk operation? I thought the wireframe just a reference.

Reply:
It actually does say to hide the `action` dropdown in the AC I think. I am more concerned about unsetting it versus having it controlled via user permissions. I think site admin users would still want the functionality available. Please confirm with Jess.